.github/workflows/test-integration-postgres-redis-rabbit.yml

```
@@ -0,0 +1,75 @@
+name: Reusable workflow for running postgres (with prisma) with rabbit and redis integration tests
```

```suggestion
name: Reusable workflow for running postgres, rabbit, and redis integration tests
```
There's nothing "Prisma" about this workflow.